__tests__/PlayButton.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen, userEvent } from '@testing-library/react-native';
+import PlayButton from '../components/PlayButton';
+
+const mockOnPress = jest.fn();
+
+const defaultProps = {
+  onPress: mockOnPress,
+  isPlayerInitialized: true,
+};
+
+const TestComponent = () => <PlayButton {...defaultProps} />;
+
+describe('PlayButton', () => {
+  test('renders correctly', () => {
+    const component = render(<TestComponent />);
+
+    // Check that the component renders without crashing
+    expect(component).toBeTruthy();
+  });
+
+  test('calls onPress when pressed', async () => {
+    const user = userEvent.setup();
+
+    render(<TestComponent />);
+
+    // Find the play button by testID
+    const playButton = screen.getByLabelText('Play Button');
+    await user.press(playButton);
+
+    expect(mockOnPress).toHaveBeenCalledTimes(1);
+  });
+
+  test('shows play icon when not playing', () => {
+    render(<TestComponent />);
+
+    // When not playing, should show play icon and not show pause icon
+    expect(screen.queryByLabelText('Play Button')).toBeTruthy();
```

Comment:
Nice that the testing gives some accessibility boosts too (playwright encourages similar)